entry

```diff
@@ -76,6 +76,18 @@ helm_repo_init() {
   fi
 }
 
+helm_content_decode() {
+  set -e
+  ENC_CHART_PATH="/chart/${NAME}.tgz.base64"
+  CHART_PATH="/${NAME}.tgz"
+  if [ ! -f "${ENC_CHART_PATH}" ]; then
+    return
```

Comment:
Should we output something to the user here?

Reply:
so basically the crd has two spec keys either `chart` or `chartContent` so this function only decodes the chartContent after it gets mounted from a configmap, so if this file doesnt exist we just process the chart spec instead